woodchipper/configs.py

```diff
@@ -60,7 +61,9 @@ class DevLogToStdout(BaseConfigClass):
         structlog.dev.set_exc_info,
         structlog.processors.UnicodeDecoder(),
         structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M.%S", utc=False),
-        structlog.processors.CallsiteParameterAdder(parameters=callsite_parameters),
+        structlog.processors.CallsiteParameterAdder(
+            parameters=callsite_parameters, additional_ignores=["woodchipper"]
+        ),
```

Comment:
The problem was that the `Exiting context` callsite was broken by the PR that added the prefix to all keys, even in single log messages and `.bind()` calls - it would list as the logger's `proxy_to_logger` function. Adding `woodchipper` to the ignored callsites puts it in the right place.